pkg/devnet/devnet.go

```diff
@@ -128,6 +128,9 @@ func runLotusMiner(ctx context.Context, home string) {
 		{"sed", "-Ei", "-e", "s/#AggregateCommits\\ =\\ true/AggregateCommits=false/",
 			filepath.Join(home, ".lotusminer", "config.toml")},
 
+		{"sed", "-Ei", "-e", "s/#EnableMarkets\\ =\\ true/EnableMarkets=false/",
```

Comment:
@dirkmc @aarshkshah1992 I think it makes sense for the `devnet` in Boost to always disable markets on Lotus.